app/assets/stylesheets/nucore.scss

```diff
@@ -628,3 +634,44 @@ li:hover {
 .nav.pull-right {
   letter-spacing: $fontSpacing;
 }
+
+
+/*
+ UCONN Redesign Table View
+*/
+td, tr, {
+  transition: ease 0.2s all;
+  border-bottom: solid  $grey 1px;
+}
+.table-striped tbody > tr:nth-child(2n+1) {
+  td {
```

Comment:
I think we should probably be consistent and use `th, td` rather than two identical blocks.